lineage/tests/test_tHMM.py

```diff
@@ -225,51 +225,51 @@ def test_getAccuracy(self):
         check_acc = all(1.0 >= x >= 0.0 for x in t.Accuracy)
         self.assertTrue(check_acc)
 
-    def test_mutual_info(self):
```

Comment:
Use `unittest.skip` rather than commenting this all out.

https://stackoverflow.com/questions/2066508/disable-individual-python-unit-tests-temporarily